services/src/unit-test/java/uk/gov/moj/sdt/services/messaging/MessageWriterTest.java

```diff
@@ -144,21 +122,47 @@ void testQueueMessage() {
      */
     @Test
     void testQueueMessageForDeadLetter() {
-        // Setup finished, now tell the mock what to expect.
-        final ISdtMessage sdtMessage = new SdtMessage();
-        sdtMessage.setSdtRequestReference("Test");
-
-        // Send the message.
-        try {
-            final Map<String, String> queueNameMap = new HashMap<>();
-            queueNameMap.put(UNIT_TEST, UNIT_TEST_QUEUE);
-
-            messageWriter.setQueueNameMap(queueNameMap);
-
-            messageWriter.queueMessage(sdtMessage, UNIT_TEST, true);
-            assertTrue(true, "Success");
-        } catch (final IllegalArgumentException e) {
-            fail("Not Expected to fail");
+            // Setup finished, now tell the mock what to expect.
+            final ISdtMessage sdtMessage = new SdtMessage();
+            sdtMessage.setSdtRequestReference("Test");
+
+            // Send the message.
+            try {
+                final Map<String, String> queueNameMap = new HashMap<>();
+                queueNameMap.put(UNIT_TEST, UNIT_TEST_QUEUE);
+
+                messageWriter.setQueueNameMap(queueNameMap);
+
+                messageWriter.queueMessage(sdtMessage, UNIT_TEST, true);
+                assertTrue(true, SUCCESS);
+            } catch (final IllegalArgumentException e) {
+                fail(NOT_EXPECTED_TO_FAIL);
+            }
+    }
+    @Test
+    void testQueueMessageForDeadLetterPerformanceLogging() {
```

Comment:
Just need to add something to this test to check what affect performance logging being switched on has.  To do this verify that mockStaticPerformanceLogger.log() has been called.